### ISSUE TITLE
Let's not use the word 'serious'

### ISSUE THREAD

**Issue by deniszgonjanin** [datacats/datacats#115]
```
----------------------------------------
DATACATS: serious problem was encountered:
   'less' command is not recognized.
              See 'datacats help' for the list of available commands
----------------------------------------
```

This makes it sound very... serious

**Comment by deniszgonjanin** [comment]
We don't want the users to panic or feel scared

**Comment by wardi** [comment]
indentation is wrong and the "DATACATS" capitalization looks wrong. I should have picked this up in review.

**Comment by JackMc** [comment]
Want me to PR this?

**Comment by JackMc** [comment]
See #116

**Comment by deniszgonjanin** [comment]
fixed via #116

**Comment by deniszgonjanin** [comment]
@JackMc lol, race condition
